lib/ruby_lsp/requests/document_link.rb

```diff
@@ -114,6 +185,29 @@ def resolve_version(uri)
 
         GEM_TO_VERSION_MAP[uri.gem_name]
       end
+
+      sig { params(node: SyntaxTree::Node, name: String).void }
+      def add_rails_document_link(node, name)
+        # ignore non-Ruby files
+        return unless @uri.match?(/\.rb$/)
+
+        if RAILTIES_VERSION && self.class.rails_documents.nil?
+          $stderr.puts "Fetching Rails Documents..."
+          self.class.fetch_rails_documents
+        end
+
+        doc_path = T.must(self.class.rails_documents)[name]
+
+        if doc_path
+          url = "#{RAILS_DOC_HOST}/v#{RAILTIES_VERSION}/#{doc_path}"
+
+          @links << LanguageServer::Protocol::Interface::DocumentLink.new(
+            range: range_from_syntax_tree_node(node),
+            target: url,
+            tooltip: "Browse the Rails documentation for: #{name}"
```

Comment:
NIT
```suggestion
            tooltip: "Jump to #{name}"
```

Reply:
I think jump is not accurate because we're not going to the source code?
And given that the implementation could give wrong suggestion, I think explicitly mentioning it's to Rails doc is still necessary.